tests/data/memgpt-0.2.11/config

```diff
@@ -31,6 +31,3 @@ path = /Users/sarahwooders/.letta
 [version]
 letta_version = 0.2.12
 
-[client]
```

Comment:
I think we want to keep this since its an example of an "old" config

Reply:
done